_test/test_utilities/test_split_vector_max_sum_p2.m

```diff
@@ -0,0 +1,198 @@
+classdef test_split_vector_max_sum_p2 < TestCase
```

Comment:
```suggestion
classdef test_split_data_blocks < TestCase
```

To make things less confusing, this file (and the test) be called `test_split_data_blocks.m` rather than `test_split_vector_max_sum_p2.m` - since it tests `split_data_blocks`.